src/Workspaces/Core/Portable/Shared/TestHooks/FeatureAttribute_Names.cs

```diff
@@ -23,6 +23,7 @@ internal partial class FeatureAttribute
         public const string GoToImplementation = nameof(GoToImplementation);
         public const string GraphProvider = nameof(GraphProvider);
         public const string InfoBar = nameof(InfoBar);
+        public const string InlineParamNameHints = nameof(InlineParamNameHints);
```

Comment:
```suggestion
        public const string InlineParameterNameHints = nameof(InlineParameterNameHints);
```

Let's be consistent with "Parameter".